packages/@okta/vuepress-site/docs/reference/api/apps/index.md

```diff
@@ -5466,6 +5466,485 @@ curl -v -X DELETE \
+                ]
+            }
+        }
+    }
+}
+```
+
+### Update Feature for application
+
+<ApiLifecycle access="ea" />
+
+<ApiOperation method="put" url="/api/v1/apps/${applicationId}/features/${featureName}" />
+
+Update a Feature for an application.
+
+#### Request parameters
+
+| Parameter       | Description                                | Parameter Type   | DataType   | Required |
+| :-------------- | :----------------------------------------- | :--------------- | :--------- | :------- |
+| applicationId   | `id` of an [app](#application-object)      | URL              | String     | TRUE     |
+| capabilities   | capabilites of the feature                  | Body              | [Capabilites Object](#capabilties-object) | TRUE     |
+| name   | name of the feature                      | URL              | String     | TRUE     |
+
+#### Reponse Parameters
+
+Updated [Application Feature](#application-feature-object).
+
+#### Request Example
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+    "create": {
+        "lifecycleCreate": {
+            "status": "ENABLED"
+        }
+    },
+    "update": {
+        "lifecycleDeactivate": {
+            "status": "ENABLED"
+        },
+        "profile":{
+            "status": "ENABLED"
+        },
+        "password":{
+            "status": "ENABLED",
+            "seed": "RANDOM",
+            "change": "CYCLE"
+        }
+    }
+}' "https://${yourOktaDomain}/api/v1/apps/${applicationId}/features/${name}"
+```
+
+This endpoint supports partial updates.
+
+```bash
+curl -v -X PUT \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+    "create": {
+        "lifecycleCreate": {
+            "status": "DISABLED"
+        }
+    }
+}' "https://${yourOktaDomain}/api/v1/apps/${applicationId}/features/${name}"
+```
+
+#### Response example
+
+```json
+{
+    "name": "USER_PROVISIONING",
+    "status": "ENABLED",
+    "description": "User provisioning settings from Okta to a downstream application",
+    "capabilities": {
+        "create": {
+            "lifecycleCreate": {
+                "status": "DISABLED"
+            }
+        },
+        "update": {
+            "profile": {
+                "status": "DISABLED"
+            },
+            "lifecycleDeactivate": {
+                "status": "DISABLED"
+            },
+            "password": {
+                "status": "DISABLED",
+                "seed": "RANDOM",
+                "change": "KEEP_EXISTING"
+            }
+        }
+    },
+    "_links": {
+        "self": {
+            "href": "http://${yourOktaDomain}/api/v1/apps/${applicationId}/features/USER_PROVISIONING",
+            "hints": {
+                "allow": [
+                    "GET",
+                    "PUT"
+                ]
+            }
+        }
+    }
+}
+```
+
 ## Models
```

Comment:
All the description fields for this table (and actually throughout the entire doc) should start with a capital letter:
> unique key for app definition => Unique key for app definition